tests/e2e/main.go

```diff
@@ -287,226 +283,132 @@ func getTestCases(selectedPredefinedTests, selectedTestFiles TestSet) (tests []t
 	return tests
 }
 
-// runs E2E tests
-// all docker containers are built sequentially to avoid race conditions when using local cosmos-sdk
-// after building docker containers, all tests are run in parallel using their respective docker containers
-func main() {
-	if err := parseArguments(); err != nil {
-		flag.Usage()
-		log.Fatalf("Error parsing command arguments %s\n", err)
-	}
-
-	testCases := getTestCases(selectedTests, selectedTestfiles)
+func deleteTargets(targets []ExecutionTarget) error {
+	var rc error = nil
```

Comment:
Nit: can be simplified to:
```suggestion

func deleteTargets(targets []ExecutionTarget) (rc error) {
```